packages/build/src/plugins/child/mutations.js

```diff
@@ -54,15 +54,33 @@ const validateReadonlyProperty = function (method, keys, proxy, key, ...args) {
   }
 
   const propName = keysA.join('.')
-  throwValidationError(`"netlifyConfig.${propName}" is read-only.`)
+  const readonlyErrorMessage = READONLY_ERROR_MESSAGES[propName] || getDefaultReadonlyError
```

Comment:
[sand] Since this variable holds a function, should it be called something like `getReadonlyErrorMessage`?

Reply:
Fixed :+1: